### PR TITLE
The file AUTHORS has been updated.

### PR DESCRIPTION
**Pull request by mamakancha**
This file was last updated in September 2016. Some more patches have been submitted by new contributors since then. This file now contains all the authors up till 16th March 2017.

### CODE REVIEW

**Comment by mamakancha** [comment]
After updating the AUTHORS, 
I got a message saying 

 `You may also want to run ./bin/mailmap_update.py to update .mailmap as well`

However, there were no changes as compared to the previous version, so n changes made there :)

**Comment by smichr** [comment]
I see duplicates -- do those still need to be added to the `.mailmap` file?

e.g. here are a few I saw right away

```
 +Val <valeriia.gladkova@gmail.com>
 +valglad <valeriia.gladkova@gmail.com>
 +szymag <szymon.mieszczak@gmail.com>
 +Szymon Mieszczak <szymon.mieszczak@gmail.com>
```

File: AUTHORS [hunk 1]
```diff
@@ -543,3 +544,72 @@ Dennis Meckel <meckel@datenschuppen.de>
...
+Rajat Thakur <Rajat Thakur>
+rajatthakur <rajatthakur1997@gmail.com>
+Dhruv Bhanushali <dhruv_b@live.com>
+geety <anjul.ten@gmail.com>
+Markus <markus.mueller.1.g@googlemail.com>
+Rishabh Madan <rishabhmadan96@gmail.com>
+Barun Parruck <barun.parruck@gmail.com>
+Bao Chau <chauquocbao0907@gmail.com>
+tanayag <tanay_agrawal@hotmail.com>
+ranjithkumar007 <ranjith.dakshana2015@gmail.com>
```
Comment (ranjithkumar007) [inline AUTHORS]: @smichr  , you can remove this redundant line [ranjithkumar007] here .. My full name is mentioned [below](https://github.com/mamakancha/sympy/blob/d0a9c81ee855ddfa46d6608f8b33db6d44cbe00f/AUTHORS#L614)

**Comment by mamakancha** [comment]
Thanks for pointing out. Will start working on this right away.

**Comment by mamakancha** [comment]
@smichr Duplicates have been removed. Please let me know if you still see any.
@ranjithkumar007 Your name has been updated as well. Thanks for your input :)

**Comment by mamakancha** [comment]
After executing `./bin/test` , I got the following error :

```
________________________________________________________________________________
______________ sympy/plotting/tests/test_plot.py:test_matplotlib _______________
  File "/home/mamakancha/sympy/sympy/plotting/tests/test_plot.py", line 282, in test_matplotlib
    plot_and_save('test')
  File "/home/mamakancha/sympy/sympy/plotting/tests/test_plot.py", line 248, in plot_and_save
    assert len(w) == 1
AssertionError
________________________________________________________________________________
__________ sympy/plotting/tests/test_plot_implicit.py:test_matplotlib __________
  File "/home/mamakancha/sympy/sympy/plotting/tests/test_plot_implicit.py", line 78, in test_matplotlib
    plot_implicit_tests('test')
  File "/home/mamakancha/sympy/sympy/plotting/tests/test_plot_implicit.py", line 64, in plot_implicit_tests
    assert len(w) == 1
AssertionError

==== tests finished: 6709 passed, 2 failed, 39 skipped, in 3432.43 seconds =====
DO *NOT* COMMIT!

```

What should be done to fix these errors? Any help is appreciated.

**Comment by mamakancha** [comment]
@isuruf Any suggestions?

File: AUTHORS [hunk 2]
```diff
@@ -543,3 +544,64 @@ Dennis Meckel <meckel@datenschuppen.de>
 Harshil Meena <harshil.7535@gmail.com>
 Micky <mickydroch@gmail.com>
 arghdos <arghdos@gmail.com>
+Jainul Vaghasia <jainulvaghasia@D-10-157-53-226.dhcp4.washington.edu>
+Michele Zaffalon <michele.zaffalon@gmail.com>
+Martha Giannoudovardi <maapxa@gmail.com>
+devangkulshreshtha <devang.kulshreshtha.cse14@itbhu.ac.in>
+steph papanik <spapanik21@gmail.com>
+Isuru Fernando <isuru.11@cse.mrt.ac.lk>
```
Comment (isuruf) [inline AUTHORS]: Duplicate with `isuruf@gmail.com`

File: AUTHORS [hunk 3]
```diff
@@ -543,3 +544,64 @@ Dennis Meckel <meckel@datenschuppen.de>
...
+msdousti <msdousti@gmail.com>
+Arif Ahmed <arif.ahmed.5.10.1995@gmail.com>
+Abdullah Javed Nesar <abduljaved1994@gmail.com>
+Lakshya Agrawal <zeeshan.lakshya@gmail.com>
+shruti <shrutishrm512@gmail.com>
+ArifAhmed1995 <arif.ahmed.5.10.1995@gmail.com>
+rohitrango <rohit.rango@gmail.com>
```
Comment (isuruf) [inline AUTHORS]: You should fix these using `.mailmap` if possible. https://github.com/rohitrango

**Comment by mamakancha** [comment]
Closing this PR to submit a new one.

**Pull request closed without merging**